Resources/Private/JavaScript/Host/Containers/ContentCanvas/InlineUI/MarkActiveNodeAsFocused/index.js

```diff
@@ -0,0 +1,32 @@
+import {Component} from 'react';
+import {connect} from 'react-redux';
+import {CR} from 'Host/Selectors/index';
+import {$transform} from 'plow-js';
+import style from './style.css';
+import {node as NodePropType} from 'Shared/PropTypes/CR/Node/index';
+
+@connect($transform({
+    focusedNode: CR.Nodes.focusedSelector
+}))
+export default class MarkActiveNodeAsFocused extends Component {
```

Comment:
On the other hand, it's not even necessary to subscribe to changes manually, you can just take the code from MarkActiveNodeAsFocused, but it into plain JS module, and call it from the render method of InlineUi. Makes sense?